tg/controllers.py

```diff
@@ -131,6 +134,17 @@ def quit(self) -> str:
     def back(self) -> str:
         return "BACK"
 
+    @bind(msg_handler, ["m"])
+    def jump_to_reply_msg(self) -> None:
+        chat_id = self.model.chats.id_by_index(self.model.current_chat)
+        if not chat_id:
+            return
+        msg = MsgProxy(self.model.current_msg)
+        if msg_id := msg.reply_msg_id:
+            if self.model.msgs.jump_to_msg_by_id(chat_id, msg_id):
+                return self.render_msgs()
+        self.present_error("Can't jump to reply msg: it's not preloaded")
```

Comment:
misleading error text. If i press `m` for message without reply in it I expect different error.